File: Python/Pi.py
def pi(n:int):
  num: float = 4.0
  den: float = 1.0
  oper: float = 1.0
  pi:float = 0.0

  for i in range(n):
    pi += oper * (num / den)
    den += 2.0
    oper *= -1.0

  return pi

print(pi(500000))

File: rust/pi.rs
fn pi(mut n: u128) -> f64 {
    let num: f64 = 4.0;
    let mut den: f64 = 1.0;
    let mut oper: f64 = 1.0;
    let mut pi:f64 = 0.0;

    loop {
        pi += oper * (num / den);
        den += 2.0;
        oper *= -1.0;
        n -= 1;
        if n == 0 {break;}
    }
    return pi
}

fn main() {
    let pi = pi(9999999);
    println!("Pi -> {}", pi);
}